README.md

```diff
@@ -1 +1,18 @@
-# MyPortfolio
\ No newline at end of file
+# Portfolio
+
+> My Portfolio
+
+## Built With
+
+- HTML
+- CSS
+
+
+
+## Authors
+
+👤 **Kareem Magdy Wilson**
+
+- GitHub: [KareemWilson](https://github.com/KareemWilson)
+- LinkedIn: [Kareem Wilson](https://www.linkedin.com/in/kareem-wilson/)
+
```

Comment:
Kindly include all the main headers of the given [README template](https://github.com/microverseinc/readme-template/blob/master/README.md). They'll make your documentation more professional.

Include the _Contributing_, _Show your support_, _Acknowledgments_, and _License_ sections of the template.